pkg/util/util.go

```diff
@@ -83,6 +83,16 @@ func GetAvailableSpace(path string) int64 {
 	return int64(stat.Bavail) * int64(stat.Bsize)
 }
 
+// GetAvailableSpaceBlock gets the amount of available space at the path specified.
+func GetAvailableSpaceBlock(path string) int64 {
```

Comment:
I would really like a some kind of test to verify this actually does what I think it does. I just made it up last Friday during the review. I don't actually know if it works.

Reply:
changed the code here

Reply:
Okay so now you switched back to the original code you said didn't work? So I am a little confused. Didn't the fstat code not work?

Reply:
yeah. dont know what did not work before. working now.